package-lock.json

```diff
@@ -202,12 +202,157 @@
       "dependencies": {
         "to-fast-properties": {
           "version": "2.0.0",
-          "resolved": "https://registry.npmjs.org/to-fast-properties/-/to-fast-properties-2.0.0.tgz",
+          "resolved": "https://artifactory.secureserver.net/artifactory/api/npm/node-virt/to-fast-properties/-/to-fast-properties-2.0.0.tgz?dl=https://registry.npmjs.org/to-fast-properties/-/to-fast-properties-2.0.0.tgz",
```

Comment:
I would encourage you to use the public npm as registry for these files.

Reply:
Fixed (by regenerating) in 124fa4a